### PR TITLE
Adjust license verbage

### PR DESCRIPTION
**Pull request by jasinner**
This includes the commits from [pull/753](https://github.com/RedHatProductSecurity/component-registry/pull/753), please review that one first.

Incorporates the feedback from CORGI-830 including:

- Use the invalid license as a license ref where possible
- Change the extracted text as specified
- Update the license list version
- Update the license comment in the packages list

@RedHatProductSecurity/corgi-devs please review

### CODE REVIEW

File: corgi/tasks/management/commands/generatemanifests.py [hunk 1]
```diff
@@ -13,17 +14,12 @@ def add_arguments(self, parser: CommandParser) -> None:
...
-            action="store_false",
-            help="Skip applying manifest fixups",
-        )
 
     def handle(self, *args, **options) -> None:
         if options["stream"]:
+            ps = ProductStream.objects.get(name=options["stream"])
             self.stdout.write(self.style.SUCCESS(f"Updating manifest for {options['stream']}"))
-            cpu_update_ps_manifest(options["stream"], fixup=options["skip_fixups"])
+            cpu_update_ps_manifest(ps.name, ps.external_name)
```
Comment (JimFuller-RedHat) [inline corgi/tasks/management/commands/generatemanifests.py:22]: for my own understanding - why do we no longer pass in fixup ?

**Review by JimFuller-RedHat — APPROVED** [review]
apart from CI failure, LGTM!

File: corgi/tasks/management/commands/generatemanifests.py (hunk 1 above)
Comment (jasinner) [reply corgi/tasks/management/commands/generatemanifests.py]: We still use some of the hardcoded values from the fixups, but they are now integrated into the ProductModel.cpes property, not just part of SBOM generation. We don't have the option to skip them anymore.

**Comment by jasinner** [comment]
Need to remove links from extracted text for license concluded fields. Just replace invalid SPDX expressions with NOASSERTION for license concluded.

**Pull request merged**